includes/acl/decompression/stream/read_stream_data.h

```diff
@@ -0,0 +1,586 @@
+#pragma once
```

Comment:
I'm not sure if it makes sense to put this under a "stream" directory since the notion of streams as used in the compression isn't present anymore. Data can and will likely be interleaved and custom per codec.

Reply:
Good point. I was trying to be symmetric with the code that writes the same compressed rotations/vectors - it's in a stream directory. I'll move this.